src/components/AdminPortal/TagActions/TagActions.spec.tsx

```
@@ -397,4 +372,4 @@ describe('Organisation Tags Page', () => {
       );
     });
   });
-});
+});
```

ESLint

[error] Insert `⏎`

(prettier/prettier)